services/housetables/src/main/java/com/linkedin/openhouse/housetables/api/validator/impl/OpenHouseUserTableHtsApiValidator.java

```diff
@@ -98,4 +98,28 @@ public void validatePutEntity(UserTable userTable) {
       throw new RequestValidationFailureException(validationFailures);
     }
   }
+
+  @Override
+  public void validateRenameEntity(UserTableKey fromUserTableKey, UserTableKey toUserTableKey) {
+    validateGetEntity(fromUserTableKey);
+    validateGetEntity(toUserTableKey);
+    List<String> validationFailures = new ArrayList<>();
+    if (fromUserTableKey.getDatabaseId().equals(toUserTableKey.getDatabaseId())
+        && fromUserTableKey.getTableId().equals(toUserTableKey.getTableId())) {
```

Comment:
Do we need to worry about the casing issue here since OH is case insensitive for the table-identifier resolution